src/SmartComponents/SystemAdvisor/index.js

```diff
@@ -1 +1,2 @@
+// ?
```

Comment:
This is vital to inventory loading atm, but I dont know why. Im not 100% sure how SystemAdvisor,Inventory, and InventoryDetail communicate